ci/cpu/libcuml/build_libcuml.sh

```diff
@@ -2,9 +2,14 @@
 
 set -e
 
-if [ "$BUILD_LIBCUML" == '1' -o "$BUILD_CUML" == '1' ]; then
-  echo "Building libcuml"
-  CUDA_REL=${CUDA_VERSION%.*}
-
-  conda build conda/recipes/libcuml --python=${PYTHON}
-fi
+if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
+  if [ "$BUILD_LIBCUML" == '1' -o "$BUILD_CUML" == '1' ]; then
+    echo "Building libcuml"
+    CUDA_REL=${CUDA_VERSION%.*}
+    conda build conda/recipes/libcuml --python=${PYTHON}
+  fi
+else
+  if [ "$BUILD_LIBCUML" == '1' ]; then
+    conda build conda/recipes/libcuml --dirty --no-remove-work-dir --python=${PYTHON}
```

Comment:
this probably will be taken care by upstream CI scripts, but if not, wouldn't it be worth keeping the `echo "Building libcuml"` for readability of logs?

Reply:
There's not much point since the job now *only* builds libcuml. It doesn't do anything else.

Reply:
Good point :)